src/D2L.Bmx/UpdateHandler.cs

```diff
@@ -61,7 +61,7 @@ internal class UpdateHandler( IGitHubClient github, IVersionProvider versionProv
 		Console.WriteLine( "Replacing the currently running BMX executable..." );
 		string currentFilePath = Environment.ProcessPath
 			?? throw new BmxException( "Failed to locate the current BMX executable." );
-		string backupPath = Path.Join( workspaceDir, $"bmx-v{localVersion}-old.bak" );
+		string backupPath = Path.Join( workspaceDir, $"bmx-{displayVersion}-old.bak" );
```

Comment:
this is also something that the user sees, so it should use the "display version"